api-reference/beta/resources/onpremisesprovisioningerror.md

```diff
@@ -0,0 +1,45 @@
+# onPremisesProvisioningError resource type
+
+Represents directory synchronization errors for users, groups, or contacts when synchronizing on-premises directories to Azure Active Directory.
```

Comment:
..., when using Microsoft synchronization product (including Azure AD Connect, DirSync and MIM + Connector).